### PR TITLE
Fix native layout landscape

### PR DESCRIPTION
**Pull request by RagnarR**
Changes:
- Added NestedScrollView in Native Ad layout.
- Added support library v4 in gradle dependencies.

### CODE REVIEW

File: library.demo/build.gradle [hunk 1]
```diff
@@ -21,8 +21,8 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-
     compile 'com.squareup.picasso:picasso:2.5.2'
+    compile 'com.android.support:support-v4:23.3.0'
```
Comment (cerberillo) [inline library.demo/build.gradle]: Can we do this without adding support libraries?

Comment (RagnarR) [inline library.demo/build.gradle]: We using an API from 10 to last one. NestedScrollView allows work with a scroll view in old APIs and in new. I prefer to use it and do not remove.

Comment (cerberillo) [inline library.demo/build.gradle]: And can't we use a relative layout that scrolls or something similar instead of a nested scrollview?

Comment (RagnarR) [inline library.demo/build.gradle]: We can, but it a bad practice :)

Comment (cerberillo) [inline library.demo/build.gradle]: why?

Comment (RagnarR) [inline library.demo/build.gradle]: better use support libraries - it allows use the same elements for each API - this what I meaned.

Comment (cerberillo) [inline library.demo/build.gradle]: I think that if you use ScrollView it's still API 1 compilant and there is no need of using not basic objects here. WDYT?

Comment (RagnarR) [inline library.demo/build.gradle]: Sure. We only need to remember, that in Android API higher 21, ScrollView could have some issues in some cases and it's all ).

Comment (cerberillo) [inline library.demo/build.gradle]: Yeah no problem, since this is a demo app, It's still ok, let's use a common ScrollView

**Comment by cerberillo** [comment]
Let's merge 👍

**Pull request merged**